src/Renderer/UI/Update.fs

```diff
@@ -506,6 +506,12 @@ let update (msg : Msg) oldModel =
         //ignore the exectue message
         else 
             model, Cmd.none
+    | SetLastViewedPropertiesTab b ->
```

Comment:
Suggest the following refactorting for Addie:
To avoid proliferating messages have a single message, and a helper:
```
UpdateModel(descr: string, updateFun: Model -> Model)

let updateModel dispatch descr updFun =
   dispatch<| UpdateModel(descr, updFun)
```
This can be called using optics to construct the update function:

```
Optic.set fieldName_ fieldValue
|> updateModel dispatch "description here"
```

Can also use `Optic.map` for a genuine update function.

Finally, from Update function (but not View function) you can and should change the model without using messages, by pipelining the Optics in the function (which returns some updated version of model.

Messages are useful to signify external events, or large updates that are done in multiple places.